apps/mi/stg/base/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ namespace: mi
 
 resources:
   - ../../base
+  - ../../base-dev
```

Comment:
This should be the same as prod

Reply:
Can remove, but it won't be 100% prod like since:
https://github.com/hmcts/sds-flux-config/pull/2226/files#diff-ff36d424c7eff5539f9f78a9f40c3f32626183ace64a93770b793c11d5f094b3R9

is already set. Without it our perms in stg would be like prod, which is very restrictive if we need to debug?

Reply:
I did not realise this , I have to say that I don't like it